### ISSUE TITLE
Accessibility: embedded iframe lacks title

### ISSUE THREAD

**Issue by kcgthb** [asciinema/asciinema-server#446]
Hi!

Accessibility checkers report that the iframe that's injected by the inline player script doesn't have a text alternative. 

> **Description**
Every inline frame (iframe) should have a text alternative that summarizes its content or purpose.
A title that summarizes the visual content will help screen reader users to understand the iframe's purpose on the page.

For instance, a generated iframe looks like:
```
<iframe src="[...]" id="[...]" name="[...]" scrolling="no" allowfullscreen="true" style="overflow: hidden; margin: 0px; border: 0px; display: inline-block; width: 100%; float: none; visibility: visible; height: 537px;"></iframe>
```

Would it be possible to add a `title` attribute to the injected iframe?

Thanks!

**Comment by kcgthb** [comment]
I think that 1. should be good enough for compliance indeed.

**Comment by ku1ik** [comment]
The fix will go out with the next asciinema server release.
